apis/field_error.go

```diff
@@ -251,6 +256,16 @@ func (fe *FieldError) normalized() []*FieldError {
 	return errors
 }
 
+// GetErrors returns the value of the nested errors.
+func (fe *FieldError) GetErrors() []FieldError {
```

Comment:
I don't love the name `GetErrors()`.  Maybe `Expand()`?

Looking at precedent, Hashicorp's `go-multierror` uses `WrappedErrors()`: https://pkg.go.dev/github.com/hashicorp/go-multierror#Error.WrappedErrors

Reply:
Done. I name it `WrappedErrors()` instead.